relay-general/tests/snapshots/test_fixtures__event_schema.snap

```diff
@@ -2222,6 +2232,14 @@ expression: "relay_general::protocol::event_json_schema()"
                 "null"
               ]
             },
+            "is_exception_group": {
+              "description": " An optional boolean value, set `true` when the exception is the platform-specific exception\n group type.  Defaults to `false`.\n\n For example, exceptions of type `ExceptionGroup` (Python), `AggregateException` (.NET), and\n `AggregateError` (JavaScript) should have `\"is_exception_group\": true`.  Other exceptions\n can omit this field.",
+              "default": null,
```

Comment:
Question: Is there a way to make the default `false`?  This really isn't a three-state value.  It's either `true` or `false`.  If omitted, should be treated as `false`.

Reply:
I do not think it's possible to set a `default` on the field at the moment. But in code you can treat empty (or `null`) field as `false`.

Reply:
I believe the event schema has some flaws, so I wouldn't worry about it too much anyway.